models/transfers/ethereum/eth/transfers_ethereum_eth_tfers.sql

```diff
@@ -0,0 +1,24 @@
+{{ config(
+    tags=['dunesql'],
+    schema = 'transfers_ethereum_eth',
+    materialized = 'incremental',
+    partition_by = ['block_month'],
+    file_format = 'delta',
+    incremental_strategy = 'merge',
+    unique_key = ['transfer_type', 'tx_hash', 'trace_address', 'wallet_address', 'block_time'], 
+    alias = alias('eth_tfers'),
```

Comment:
This should be just `eth` right? So we'd have a table show up as `transfers_ethereum.eth` on Dune

Reply:
hmm there's a transfers_ethereum_eth.sql model that @MSilb7 built... It's a different set up and has different columns as the table isn't deduped like we have for balances. 

That's why I added tfers to the end of the table.

Reply:
Ok we should converge to this new design if it makes more sense, I'll reach out to @MSilb7 to get his thoughts